### PR TITLE
[job submission] Move job_manager to dashboard module, common parts to common.py

### PR DESCRIPTION
**Pull request by edoakes**
## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by edoakes** [comment]
@architkulkarni stamp please for codeowner!

**Review by architkulkarni — APPROVED** [review]
Were there any code changes besides moving things around?

**Comment by edoakes** [comment]
@architkulkarni no, just renamed a constant

File: python/ray/tests/BUILD [hunk 1]
```diff
@@ -306,14 +306,6 @@ py_test(
     deps = ["//:ray_lib"],
 )
 
-py_test(
```
Comment (jiaodong) [inline python/ray/tests/BUILD:309]: where are we running job manager test on CI after this move ?

Comment (edoakes) [reply python/ray/tests/BUILD:309]: it gets run in the dashboard tests (there's a wildcard there that runs all tests in subdir)

Comment (edoakes) [reply python/ray/tests/BUILD:309]: https://buildkite.com/ray-project/ray-builders-pr/builds/18488#8a6f5be9-5bee-4820-88b4-472a8fccbe51/461-3399

**Pull request merged**